reference/app-availability.html.md.erb

```diff
@@ -0,0 +1,124 @@
+---
+title: App Availability and Resiliency
+layout: docs
+sitemap: false
+nav: firecracker
+---
+
+A resilient app is resistant to events like hardware failures or outages. Redundancy, supported by monitoring and scaling, is one of the ways to make your app more resilient, and we provide some built-in features to make this easier.
+
+But first, it's important to understand what your app needs to take care of.
+
+## What your app needs to do
+
+<div class="callout">
+<p><b>A note about Fly Proxy</b>: 
+Fly Proxy does a lot of work in the background, applying handlers, managing traffic and load balancing, and managing connections to services. Fly Proxy also automatically starts and stops Machines based on traffic to your app. The proxy works for [process groups with services defined](/docs/getting-started/app-services/#services-routed-with-fly-proxy).
+</div>
```

Comment:
A callout to me feels like a deeper dive, something extra that needs to be explained. That's why I'd suggest moving the Fly Proxy callout to L23. This way, you talk about the Fly Proxy stuff that's directly related to this doc and after that show the callout that explains the Fly Proxy further. This is very much an opinion so feel free to ignore.

Reply:
@Johannes-Werbrouck thanks for this. I've moved this bit around a few times! Wasn't entirely happy with where it ended up. :) I'll give your idea a try.